CHANGELOG.md

```diff
@@ -0,0 +1,101 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+-
+
+## [v0.6.0-rc1] - 2018-10-16
+
+### Fixed
+- Reorganized and cleaned up repository
+- Fixed examples throughout
+
+### Added
+- **Collections added**: Collections are a type of Catalog with additional fields, such as provider and license. Items must belong to a single Collection.
+- **Extension maturity**: Protocol for providing maturity classification for extensions based on stability and implementations.
+- **Commons extension**: The previous 'Collections' extension is now the 'Commons' extension and allows an Item to inherit properties from it's Collection.
+- **Datetime-range extension**: Extension for providing start and end datetimes.
+- **Scientific extension**: Extension for providings links to scientific publications relating to the data.
+- **rel types**: A list of supported 'rel' types are provided for use when specifying links.
+- **eo:constellaion**: A new field in the EO specification to specify a grouping of platforms.
+- **stac_version**: The stac_version field is required on all Catalogs (and Collections)
+
+### Changed
+- **eo:bands**: The eo:bands field is now an array rather than a dictionary, and has been moved inside of 'properties' in a STAC Item.
```

Comment:
To "Changed"I would add:
*  **links**: The links fields are now an array rather than a dictionary.
* **properties**: Fields with the data type array or objects are allowed inside the `properties` in a STAC Item.
* **description**: Description fields now allow formatting with CommonMark.

The asset schema has also changed (I think: name => title, mime_type => type, added fields)